src/apps/frontend/contexts/auth.provider.tsx

```diff
@@ -37,6 +41,17 @@ const loginFn = async (
   return result;
 };
 
+const signupFn = async (
+  username: string,
+  password: string,
+): Promise<ApiResponse<AccessToken>> => {
+  const result = await authService.signup(username, password);
+  if (result.data) {
+    localStorage.setItem('access-token', JSON.stringify(result.data));
```

Comment:
I don't think we want to do this as part of sign up process. User should be redirected to sign in page (which you are doing) but we should not store anything in local storage.